plexapi/library.py

```diff
@@ -2849,6 +2861,133 @@ def _loadData(self, data):
+        self.title = data.attrib.get('title')
+        self._promoted = True  # flag to indicate if this hub has been promoted on the list of managed recommendations
+
+        parent = self._parent()
+        self.librarySectionID = parent.key if isinstance(parent, LibrarySection) else parent.librarySectionID
+
+    def reload(self):
+        """ Reload the data for this managed hub. """
+        key = f'/hubs/sections/{self.librarySectionID}/manage'
+        hub = self.fetchItem(key, self.__class__, identifier=self.identifier)
+        self.__dict__.update(hub.__dict__)
+        return self
+
+    def move(self, after=None):
+        """ Move a managed hub to a new position in the library's Managed Recommendations.
+
+            Parameters:
+                after (obj): :class:`~plexapi.library.ManagedHub` object to move the item after in the collection.
+
+            Raises:
+                :class:`plexapi.exceptions.BadRequest`: When trying to move a Hub that is not a Managed Recommendation.
+        """
+        if not self._promoted:
+            raise BadRequest('Collection must be a Managed Recommendation to be moved')
+        key = f'/hubs/sections/{self.librarySectionID}/manage/{self.identifier}/move'
+        if after:
+            key = f'{key}?after={after.identifier}'
+        self._server.query(key, method=self._server._session.put)
+
+    def remove(self):
+        """ Removes a managed hub from the library's Managed Recommendations.
+
+            Raises:
+                :class:`plexapi.exceptions.BadRequest`: When trying to remove a Hub that is not a Managed Recommendation
+                    or when the Hub cannot be removed.
+        """
+        if not self._promoted:
+            raise BadRequest('Collection must be a Managed Recommendation to be removed')
+        if not self.deletable:
+            raise BadRequest(f'{self.title} managed hub cannot be removed' % self.title)
+        key = f'/hubs/sections/{self.librarySectionID}/manage/{self.identifier}'
+        self._server.query(key, method=self._server._session.delete)
+
+    def updateVisibility(self, recommended=None, home=None, shared=None):
+        """ Update the managed hub's visibility settings.
+
+            Parameters:
+                recommended (bool): True to make visible on your Library Recommended page. False to hide. Default None.
+                home (bool): True to make visible on your Home page. False to hide. Default None.
+                shared (bool): True to make visible on your Friends' Home page. False to hide. Default None.
+
+            Example:
+
+                .. code-block:: python
+
+                    managedHub.updateVisibility(recommended=True, home=True, shared=False).reload()
+                    # or using chained methods
+                    managedHub.promoteRecommended().promoteHome().demoteShared().reload()
+        """
+        params = {}
```

Comment:
Changing the visibility of the default library hubs actually fails unless all three parameters are provided. Set the default parameters to the current attribute values.
```suggestion
        params = {
            'promotedToRecommended': int(self.promotedToRecommended),
            'promotedToOwnHome': int(self.promotedToOwnHome),
            'promotedToSharedHome': int(self.promotedToSharedHome),
        }
```